GetMessage/userPreferenceChecker/__tests__/messageReadStatusAuth.test.ts

```diff
@@ -91,6 +91,23 @@ describe("canAccessMessageReadStatus |> ok", () => {
     expect(mockServicePreferencesFind).not.toHaveBeenCalled();
   });
 
+  it("should return false if profile servicePreferencesSettings is of type LEGACY (version is -1)", async () => {
```

Comment:
There is something with this test, it ends with success even if we pass a servicePreferenceSettings not LEGACY

Reply:
fixed in https://github.com/pagopa/io-functions-services/pull/309/commits/556d47841a85139347a042991bee29c38abf78e9